internal/dockerfile/dockerfile_test.go

```diff
@@ -117,3 +117,20 @@ func TestNoAddsToNoMounts(t *testing.T) {
 	}
 	assert.Empty(t, mounts)
 }
+
+func TestFromImage(t *testing.T) {
+	df := Dockerfile(`FROM gcr.io/image-a`)
+	images, err := df.FindImages()
+	assert.NoError(t, err)
+	if assert.Equal(t, 1, len(images)) {
+		assert.Equal(t, "gcr.io/image-a", images[0].String())
+	}
+}
+
+func TestFromImageMalformed(t *testing.T) {
```

Comment:
Maybe add a test for a Dockerfile with a malformed FROM?  Like:

```Dockerfile
FROM
```

or

```Dockerfile
FROM foo bar
```

Also do we want to support this?
```Dockerfile
FROM foo AS bar
```

Reply:
added some more tests!

Reply:
Awesome, thanks!